### PR TITLE
Disable Commit trigger for certain Jenkins precommit

### PR DESCRIPTION
**Pull request by Abacn**
**Please** add a meaningful description for your change here

Commit trigger for the following precommit are disabled, as same GitHub Action workflows are running

| Name | Jenkins status | GitHub Action status |
|-----|-----|-----|
| PreCommit_PythonDocs | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonDocs_Cron/) | [.github/workflows/beam_PreCommit_PythonDocs.yml](https://github.com/apache/beam/actions/workflows/beam_PreCommit_PythonDocs.yml)
| PreCommit_PythonLint | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_PythonLint_Cron/) | [.github/workflows/beam_PreCommit_PythonLint.yml](https://github.com/apache/beam/actions/workflows/beam_PreCommit_PythonLint.yml)
| PreCommit_RAT | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_RAT_Cron/) | [.github/workflows/beam_PreCommit_RAT.yml](https://github.com/apache/beam/actions/workflows/beam_PreCommit_RAT.yml)
| PreCommit_Spotless | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Spotless_Cron/) | [.github/workflows/beam_PreCommit_Spotless.yml](https://github.com/apache/beam/actions/workflows/beam_PreCommit_Spotless.yml)
| PreCommit_Website | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Website_Cron/) | [.github/workflows/beam_PreCommit_Website.yml](https://github.com/apache/beam/actions/workflows/beam_PreCommit_Website.yml)
| PreCommit_Website_Stage_GCS | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Website_Stage_GCS_Cron/) | [PreCommit Website Stage GCS](https://github.com/apache/beam/actions/workflows/beam_PreCommit_Website_Stage_GCS.yml)
| PreCommit_Whitespace | [Build Status](https://ci-beam.apache.org/job/beam_PreCommit_Whitespace_Cron/) | [.github/workflows/beam_PreCommit_Whitespace.yml](https://github.com/apache/beam/actions/workflows/beam_PreCommit_Whitespace.yml)

cron jobs and phrase trigger are unaffected

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by Abacn** [comment]
Run seed job

File: .test-infra/jenkins/job_PreCommit_PythonDocs.groovy [hunk 1]
```diff
@@ -26,7 +26,8 @@ PrecommitJobBuilder builder = new PrecommitJobBuilder(
     timeoutMins: 30,
     triggerPathPatterns: [
       '^sdks/python/.*$',
-    ]
+    ],
+    commitTriggering: false
```
Comment (damccorm) [inline .test-infra/jenkins/job_PreCommit_PythonDocs.groovy:30]: Can we just remove these entirely? We've been a good state for a while, and I don't think keeping them is getting us much. We can always revert if there's a problem (which is what we'd need to do anyways)

Comment (Abacn) [reply .test-infra/jenkins/job_PreCommit_PythonDocs.groovy]: Sounds good, done

note that left jenkins/README.md as is for now

**Comment by Abacn** [comment]
Found the phrase trigger does not work well. There are at least following problem

- Another comment, regardless what content it is, will cancel the ongoing phrase triggered run for another job

- If not triggered by commit, phrase triggered job won't update its status to the PR

**Comment by Abacn** [comment]
Run Whitespace PreCommit

**Comment by Abacn** [comment]
Run PythonDocs PreCommit

**Comment by Abacn** [comment]
- whitespace precommit is cancelled https://github.com/apache/beam/actions/runs/6090179476/job/16524631139

- python doc precommit will be the same once this comment sent out https://github.com/apache/beam/actions/runs/6090182277/job/16524662987

**Comment by damccorm** [comment]
> If not triggered by commit, phrase triggered job won't update its status to the PR

This is probably not an issue for this set of jobs.

> Another comment, regardless what content it is, will cancel the ongoing phrase triggered run for another job

I think blocking this pr on this probably makes sense

**Comment by Abacn** [comment]
Run RAT PreCommit

**Comment by Abacn** [comment]
irrelevant comment

**Comment by Abacn** [comment]
Run PythonDocs PreCommit

**Comment by Abacn** [comment]
Run Whitespace PreCommit

File: .github/workflows/beam_PreCommit_Whitespace.yml [hunk 2]
```diff
@@ -19,10 +19,10 @@ on:
   push:
     tags: ['v*']
     branches: ['master', 'release-*']
-    paths: ['*.md', '*.build.gradle','.github/workflows/beam_PreCommit_Whitespace.yml']
+    paths: ['*.md', '*.gradle', '*.gradle.kts', '.github/workflows/beam_PreCommit_Whitespace.yml']
```
Comment (Abacn) [inline .github/workflows/beam_PreCommit_Whitespace.yml:22]: This should in sync with https://github.com/apache/beam/blob/df0c25d5c315baa880355c4cf5499135f1aa4998/sdks/python/test-suites/tox/py38/build.gradle#L152

**Pull request merged**